wp-dependency-installer.php

```diff
@@ -123,14 +123,16 @@ public function run( $plugin_path ) {
 		}
 
 		/**
-		 * Register data from wp-dependencies.json file.
+		 * Decode json config data from a file.
+		 *
+		 * @param string json file path.
 		 *
 		 * @return bool|array $config
 		 */
-		private function register_json_file( $plugin_path ) {
+		public function json_file_decode( $json_file ) {
```

Comment:
Is there some reason to make the scope public?

Reply:
Devs may also want to define a different folder for the `wp-dependencies.js` file

Reply:
That’s not how we’ve ever defined $plugin_path. Being vague as to the location likely will end with lots of confusion and questions from inexperienced devs.

Reply:
> That’s not how we’ve ever defined $plugin_path. Being vague as to the location likely will end with lots of confusion and questions from inexperienced devs.

For this reason within README file I have kept only one possible way of defining dependencies ( ie. same DIR `wp-depencies.json` )

I renamed this function from `register_json_file` to `json_file_decode` just only to allow a little more flexibility and readability of current code.

An example use case:

```php
$wpdi   = WP_Dependency_Installer::instance();

$config = $wpdi->json_file_decode( < path to alternative wp-depencies.json );

$wpdi->register( $config, __DIR__ );
$wpdi->run( __DIR__ );
```